examples/tts/conf/fastpitch_speaker_adaptation.yaml

```diff
@@ -0,0 +1,303 @@
+# This config contains the default values for training FastPitch model with aligner using 44.1KHz sampling 
```

Comment:
This PR is quite substantial. We could improve on the following

- Consider creating a new class, subclass, mixin, etc to house this work
- This new object should contain all needed docstrings which are currently in the yaml file
- We should have two yamls: 1 for gst, and 1 for SV embeddings
- The tutorial is ok for NeMo, but it would be nice to make it easier to understand at parts
  - Parts of the tutorial that are function definitions can be moved into NeMo itself with a link to the github code and the corresponding cells that call it can be replaced with a simple function/script call